### ISSUE TITLE
 No toolbar icon to be found anywhere

### ISSUE THREAD

**Issue by beherit** [inbasic/ignotifier#384]
Problem described in https://github.com/inbasic/ignotifier/issues/323 appeared again after install version 0.6.2.

**Comment by inbasic** [comment]
So you can see the button on a clean profile but after a few restarts it just disappears?

**Comment by beherit** [comment]
No, now I don't see the button on dirty profile after upgrade to version 0.6.2, when I install previous version everything works.

**Comment by inbasic** [comment]
How about a new profile?

To debug the problem on your current profile, open the error console (Ctrl + Shift +J). Clear the console and and disable and re-enable the extension. See if you get an error in the console.

**Comment by boyssan** [comment]
[](url)

**Comment by boyssan** [comment]
#__

**Comment by beherit** [comment]
Sory but I don't have time to test in on new profile - I have a lot things to synchronize. What is important: problem exists only on my Dell with Windows 7, on other machine everything works. Problem disappear when I turn off this extension https://addons.mozilla.org/pl/firefox/addon/starbuttoninurlbar/ but on working profile I have this extension too...

Log from console:
```
Could not read chrome manifest 'file:///C:/Program%20Files/Mozilla%20Firefox/chrome.manifest'.
Warning: attempting to write 4192 bytes to preference extensions.bootstrappedAddons. This is bad for general performance and memory usage. Such an amount of data should rather be written to an external file.
unreachable code after return statement bootstrap.js:1982:2
Warning: attempting to write 4192 bytes to preference extensions.bootstrappedAddons. This is bad for general performance and memory usage. Such an amount of data should rather be written to an external file.
Warning: attempting to write 4192 bytes to preference extensions.bootstrappedAddons. This is bad for general performance and memory usage. Such an amount of data should rather be written to an external file.
SSL Observatory WARN: Failed to initialize NSS component:Error: couldn't find function symbol in library
While creating services from category 'profile-after-change', could not create service for entry 'Sapi Speech Synth', contract ID '@mozilla.org/synthsapi;1'
mutating the [[Prototype]] of an object will cause your code to run very slowly; instead create the object with the correct initial [[Prototype]] value using Object.create Preferences.js:467:1
[CustomizableUI] NotFoundError: Node was not found -- :1155 CustomizableUI.jsm:166
InvalidStateError: An attempt was made to use an object that is not, or is no longer, usable WindowsPreviewPerTab.jsm:359:0
Strona używa certyfikatu SHA-1 - zalecane jest używanie certyfikatów z algorytmami sygnatur wykorzystującymi funkcje skracania silniejsze niż SHA-1.[Więcej informacji] otrs.allegrogroup.com
Using //@ to indicate sourceMappingURL pragmas is deprecated. Use //# instead jquery-2.0.3.min.js:1:0
Metoda getAttributeNode() jest przestarzała. Należy korzystać z getAttribute(). desktop_module_main.js:86:152
```
What is very strange - the icon appear when I opened new window! Unfortunately after restart Firefox icon disappear again.

**Comment by boyssan** [comment]
Could not read chrome manifest 'file:///C:/Program%20Files/Mozilla%20Firefox/chrome.manifest'.
Warning: attempting to write 4192 bytes to preference extensions.bootstrappedAddons. This is bad for general performance and memory usage. Such an amount of data should rather be written to an external file.
unreachable code after return statement bootstrap.js:1982:2
Warning: attempting to write 4192 bytes to preference extensions.bootstrappedAddons. This is bad for general performance and memory usage. Such an amount of data should rather be written to an external file.
Warning: attempting to write 4192 bytes to preference extensions.bootstrappedAddons. This is bad for general performance and memory usage. Such an amount of data should rather be written to an external file.
SSL Observatory WARN: Failed to initialize NSS component:Error: couldn't find function symbol in library
While creating services from category 'profile-after-change', could not create service for entry 'Sapi Speech Synth', contract ID '@mozilla.org/synthsapi;1'
mutating the [[Prototype]] of an object will cause your code to run very slowly; instead create the object with the correct initial [[Prototype]] value using Object.create Preferences.js:467:1
[CustomizableUI] NotFoundError: Node was not found -- :1155 CustomizableUI.jsm:166
InvalidStateError: An attempt was made to use an object that is not, or is no longer, usable WindowsPreviewPerTab.jsm:359:0
Strona używa certyfikatu SHA-1 - zalecane jest używanie certyfikatów z algorytmami sygnatur wykorzystującymi funkcje skracania silniejsze niż SHA-1.[Więcej informacji] otrs.allegrogroup.com
Using //@ to indicate sourceMappingURL pragmas is deprecated. Use //# instead jquery-2.0.3.min.js:1:0
Metoda getAttributeNode() jest przestarzała. Należy korzystać z getAttribute(). desktop_module_main.js:86:152

**Comment by beherit** [comment]
@inbasic any solution?

**Comment by beherit** [comment]
??

**Comment by inbasic** [comment]
Well, since you have many extensions, it is really hard to catch the error. All these errors are related to the other extensions.

Try to disabling the notifier. Now clear the console and re-enable the notifier. see if you can find any relevant error in the console.

**Comment by Izkata** [comment]
A thought:  I use Classic Theme Restorer, which gives me an addon bar at the bottom of the page.  This is where I normally keep the Gmail Notifier, but last week when I finally upgraded Firefox and a bunch of addons, the notifier disappeared.

I eventually found it on the navigation bar, all the way to the right, along with a few other new icons hidden behind a dropdown - your suggestion about a clean profile also put it there, but without the extra icons so I could see where it went.

As far as I remember, this is the only one I use that moved from the addon bar to somewhere else.
